scripts/fonts.js

```diff
@@ -0,0 +1,143 @@
+// read font file
+import { Font } from "fonteditor-core";
+import fs from "fs";
+import { execSync } from "child_process";
+import request from "sync-request";
+
+function createRangeArray(start, end) {
+  return Array.from({ length: end - start + 1 }, (_, i) => start + i);
+}
+
+function loadGoogleFonts(fontSpec, destFolder) {
+  for (const fontFamily in fontSpec) {
+    const downloadLockFile = `${downloadLockFolder}/${fontFamily}.lock`;
```

Comment:
This feels like working around a shortcoming in this library, that it knows how to compute the destination filename but it won't do so until you ask it to actually download the font. I guess it works, though.

Reply:
oh, I suppose I could do something like strip out the spaces and check for matching files now that I look at this

Reply:
Oh you mean like not using the Google font Downloader at all

Reply:
Okay, fixed in 854ec9e.